kitsune/users/templates/users/mobile/profile.html

```diff
@@ -80,6 +80,10 @@
           <header>{{ _('Facebook') }}</header>
           <a class="detail" rel="nofollow" href="{{ profile.facebook }}">{{ profile.facebook }}</a>
         {% endif %}
+        {% if profile.mozillians %}
+          <header>{{ _('Mozillians') }}</header>
+          <a class="detail" rel="nofollow" href="http://mozillians.org/u/{{profile.mozillians}}">{{ profile.mozillians }}</a>
```

Comment:
Also, it looks like they like the trailing slash: https://mozillians.org/en-US/u/safwan/

Add a `/` to the end and save another redirect.